app/admin/dashboard.rb

```diff
@@ -2,10 +2,22 @@
   menu priority: 1, label: proc { I18n.t("active_admin.dashboard") }
 
   content title: proc { I18n.t("active_admin.dashboard") } do
-    div class: "blank_slate_container", id: "dashboard_default_message" do
-      span class: "blank_slate" do
-        span I18n.t("active_admin.dashboard_welcome.welcome")
-        small I18n.t("active_admin.dashboard_welcome.call_to_action")
+
+    section "Background Jobs" do
```

Comment:
maybe we should create a little feature spec for this? like a smoke test that the admin dashboard is up and running when visited